src/data/polynomial/coeff.lean

```diff
@@ -240,8 +237,10 @@ end
 lemma mul_X_injective : function.injective (λ P : R[X], X * P) :=
 pow_one (X : R[X]) ▸ mul_X_pow_injective 1
 
-lemma C_mul_X_pow_eq_monomial (c : R) (n : ℕ) : C c * X^n = monomial n c :=
-monomial_eq_C_mul_X.symm
+lemma C_mul_X_eq_monomial (c : R) (n : ℕ) : C c * X = monomial 1 c := monomial_eq_C_mul_X.symm
+
+lemma C_mul_X_pow_eq_monomial (c : R) (n : ℕ) : C c * X ^ n = monomial n c :=
+monomial_eq_C_mul_X_pow.symm
```

Comment:
I think we should take the opportunity to standardise on a direction for this lemma. I count 18 usages of `C_mul_X_pow_eq_monomial`, all of them rewrites, 4 of which are backwards rewrites, and 33 usages of `monomial_eq_C_mul_X[_pow]`, all rewrites except 2, 13 of which are backwards rewrites and 1 of which is `monomial_eq_C_mul_X.symm`.

Since in total we have 24 usages in the direction of `monomial_eq_C_mul_X_pow`, more than the 17 in the direction `C_mul_X_pow_eq_monomial`, I suggest deleting `_eq_monomial` and using `monomial_eq`.


```suggestion
```

Reply:
Good point! I count 33 usages of `X_pow_eq_monomial`. Looks like we have one item to complete the `monomial_eq` and `eq_monomial` collection: [`polynomial.monomial_eq_smul_X`](https://leanprover-community.github.io/mathlib_docs/data/polynomial/basic.html#polynomial.monomial_eq_smul_X) (8 forward rewrites).

I'd say it's best to pick a consistent direction and change all the lemmas to follow that. I'll let you choose which one, since I only have a mild preference towards the majority direction, `eq_monomial`.